scripts/install_backend_python_libs_test.py

```diff
@@ -689,7 +679,10 @@ def test_pip_install_with_import_error_and_darwin_os(self):
             sys.modules['pip'] = None
             with os_name_swap, self.print_swap, self.swap_check_call:
                 with self.assertRaisesRegexp(
-                    ImportError, 'Error importing pip: No module named pip'):
+                    ImportError,
+                    'Error importing pip: import of pip halted; '
```

Comment:
This error message seems a bit opaque. Can it be clarified?

Ditto below.

(Or is it just the py3 version of "No module named pip"?)

Reply:
Yes, this is an internal Python error.